project.yaml

```diff
@@ -63,6 +63,13 @@ dataSources:
           kind: cosmos/EventHandler
           filter:
             type: "withdraw_rewards"
+        - handler: handleCW20Transfer
+          kind: cosmos/EventHandler
```

Comment:
why an EventHandler here instead of a MessageHandler? Looking at the js hook seems it doesn't need the event at all. Mostly asking because I believe there are potentially N events per message, so it might be more efficient to match the handler at the message level rather than having to lookup each of its events.

Reply:
I was discussing this with Bryan and we settled upon the conclusion that only handling known successful state changes instead of each and every message will likely perform better - instead of dealing with failed txs and having to crash out from a check. What are your thoughts?

Reply:
make sense. Probably want to have that on the `handleLegacyBridgeSwap` then so it's consistent.